README.rst

```diff
@@ -154,10 +154,7 @@ Public Key Authentication
 .. code-block:: python
 
    session.userauth_publickey_fromfile(
-       username, 'my_pkey.pub', 'my_pkey', '')
-
-
-Where ``''`` can be a passphrase.
+       username, 'pkey', 'optional pkey.pub' , 'optional passphrase')
```

Comment:
This can just be 
```
username, 'pkey'`
```

[The documentation](https://ssh2-python.readthedocs.io/en/latest/session.html#ssh2.session.Session.userauth_publickey_fromfile) has the optional arguments for passphrase and publickey.